libs/application-generic/src/usecases/get-subscriber-template-preference/get-subscriber-template-preference.command.ts

```diff
@@ -1,7 +1,6 @@
 import { NotificationTemplateEntity, SubscriberEntity } from '@novu/dal';
-import { IsDefined, IsNotEmpty, IsOptional } from 'class-validator';
-
 import { ITenantDefine } from '@novu/shared';
+import { IsDefined, IsNotEmpty, IsOptional } from 'class-validator';
```

Comment:
Can we revert the whitespace changes here and on the below file to keep the PR tidy?